include/mqtt/client.hpp

```diff
@@ -1054,7 +1054,7 @@ class client : public endpoint<std::mutex, std::lock_guard, PacketIdBytes> {
         any session_life_keeper,
         boost::system::error_code& ec) {
         start_session(force_move(props), force_move(session_life_keeper));
-        ec = boost::system::errc::make_error_code(boost::system::errc::success);
+        ec = {};
```

Comment:
It is difficult to understand. Could you revert it as the original code?

Reply:
Reverted